tools/cli.js

```diff
@@ -24,14 +24,64 @@ const log = (msg) => {
   if (userInput) rl.write(userInput);
 };
 
+function completeByFields(input, object) {
+  return Object.keys(object)
+    .filter(c => !c.startsWith('_') && (!input || c.startsWith(input)));
```

Comment:
@aqrln the problem is it can be undefined when you press tab with no input, not sure how to handle this better